apps/backend/src/modules/vendors/libs/maps/udemy-course-fields-map.map.ts

```diff
@@ -1,12 +1,12 @@
 import { CourseField } from "~/modules/vendors/libs/enums/enums.js";
 import { type CourseFieldsMapping } from "~/modules/vendors/libs/types/types.js";
 
-const UdemyCourseFieldsMapping: CourseFieldsMapping = {
+const UdemyCourseFieldsMap: CourseFieldsMapping = {
```

Comment:
```suggestion
const UdemyCourseFieldsMap: CourseFieldsMapping = {
```
> A map is a very useful and frequently used data structure in any programming language. For maps in the JavaScript world, there is a specific naming convention. The name should follow the pattern aToB, where a serves as the key for retrieving values from the map, followed by the preposition To, implying the mapping of two things, and then B, which is the mapped value for a.

https://dev.to/what1s1ove/javascript-naming-conventions-are-important-45n

Reply:
I saw two patterns in the first article: 1) xToY and 2) xMap. I chose the second one because the name UdemyCourseSectionFieldsToCourseSectionTitle seemed too long to me. 
Thanks for the article!